### PR TITLE
added psutil python package

### PR DESCRIPTION
**Pull request by lee218llnl**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-psutil/package.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+class PyPsutil(Package):
+    """psutil is a cross-platform library for retrieving information on
+    running processes and system utilization (CPU, memory, disks, network)
+    in Python."""
+
+    homepage = "https://pypi.python.org/pypi/psutil"
+    url      = "https://pypi.python.org/packages/d9/c8/8c7a2ab8ec108ba9ab9a4762c5a0d67c283d41b13b5ce46be81fdcae3656/psutil-5.0.1.tar.gz#md5=153dc8be94badc4072016ceeac7808dc"
+
+    version('5.0.1', '153dc8be94badc4072016ceeac7808dc')
+
```
Comment (citibeth) [inline var/spack/repos/builtin/packages/py-psutil/package.py]: Please change to:
```
   version('5.0.1', '153dc8be94badc4072016ceeac7808dc',
        url      = "https://pypi.python.org/packages/d9/c8/8c7a2ab8ec108ba9ab9a4762c5a0d67c283d41b13b5ce46be81fdcae3656/psutil-5.0.1.tar.gz#md5=153dc8be94badc4072016ceeac7808dc")
```

I believe there is no need for a separate `url=` here.

@adamjstewart Should this use `PythonPackage`?

Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-psutil/package.py]: Not until `PythonPackage` is merged 😄

File: var/spack/repos/builtin/packages/py-psutil/package.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    extends('python')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-psutil/package.py]: Add:
```python
depends_on('python@2.6:')
```

Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-psutil/package.py]: Might also want setuptools.

**Pull request merged**